app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Also Like should belong to author not user i.e `belongs_to :author, class_name: 'User'`